core/capabilities/ccip/ccipevm/helpers.go

```diff
@@ -31,3 +33,13 @@ func decodeExtraArgsV1V2(extraArgs []byte) (gasLimit *big.Int, err error) {
 	}
 	return ifaces[0].(*big.Int), nil
 }
+
+// abiEncodeMethodInputs encodes the inputs for a method call.
+// example abi: `[{ "name" : "method", "type": "function", "inputs": [{"name": "a", "type": "uint256"}]}]`
+func abiEncodeMethodInputs(abiDef abi.ABI, inputs ...interface{}) ([]byte, error) {
+	packed, err := abiDef.Pack("method", inputs...)
```

Comment:
I see this only used in one place and the method is indeed called `method`, so thats why this worse. Might be worth checking this with a test, but I feel like this will fail if there is no method called `method` in the ABI

Reply:
> or is that not the case in this particular scenario?

It's not the case,this is similar to https://github.com/smartcontractkit/chainlink/blob/e37f7e2d1a1859a0853ad6ecfbe3c339c02d2248/core/chains/evm/utils/ethabi.go#L34-L33 but more performant, since it doesn't parse the abi on each call. That's why i didn't re-use what we have.